t/unit/conftest.py

```diff
@@ -0,0 +1,78 @@
+from unittest.mock import MagicMock
+import pytest
+
+sentinel = object()
+
+class _patching(object):
```

Comment:
what object is this?

Reply:
It is used as default value in methods of `_patching` class. BTW it is just copy-and-paste from Case library...